/home/hamdiaj/semantic-robustness/scale_robustness.py:717: RuntimeWarning: invalid value encountered in multiply
  self.corners_matrix = np.ones([self.two_to_n,1]) @ np.expand_dims(self.a,axis=0) + self.mask.T * (np.ones([self.two_to_n,1]) @ np.expand_dims(self.r,axis=0))
/home/hamdiaj/semantic-robustness/scale_robustness.py:824: RuntimeWarning: invalid value encountered in multiply
  temp = [my_region.r[ii] * ((my_region.mask_c[ii,::] - my_region.mask[ii,::])* my_region.mask_c[kk,::]).reshape(1,-1) @ G_D[::,ii].reshape(-1,1) for ii in range(my_region.n) if ii != kk]
/home/hamdiaj/semantic-robustness/scale_robustness.py:826: RuntimeWarning: invalid value encountered in multiply
  temp = [my_region.r[ii] * ((my_region.mask[ii,::] - my_region.mask_c[ii,::])* my_region.mask[kk,::]).reshape(1,-1) @ G_D[::,ii].reshape(-1,1) for ii in range(my_region.n) if ii != kk]
